### ISSUE TITLE
[BUG] Custom fonts don't get displayed in the trace-viewer

### ISSUE THREAD

**Issue by mxschmitt** [microsoft/playwright#9173]
**Context:**
- Playwright Version: ToT

**Code Snippet**

```html
<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.1.0/css/all.css" />
</head>

<body>
    <i class="fab fa-microsoft"></i>
</body>

</html>
```

```javascript


const { chromium } = require('playwright');

(async () => {
  const browser = await chromium.launch({
    headless: false
  });
  const context = await browser.newContext();
  await context.tracing.start({screenshots: true, snapshots: true})

  const page = await context.newPage();

  await page.goto('http://localhost:5000/test');

  await page.click('body');
  await context.tracing.stop({path: "my-trace.zip"})
  // ---------------------
  await context.close();
  await browser.close();
})();
```

**Describe the bug**

Expected: Icon gets shown
Actual: 
![image](https://user-images.githubusercontent.com/17984549/134898436-91e6541d-b91c-449f-b4b8-64025b097d74.png)

Investigation:

1. `curl -v https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/webfonts/fa-brands-400.woff2` returns `content-type: application/octet-stream; charset=utf-8`
2. CDP tries to return it as text over the wire since it contains UTF-8
3. While sending over the wire it gets corrupted
4. Chromium can't parse the file when we build again the site together and writes it inside their logs that the font is corrupted

See https://chromium-review.googlesource.com/c/chromium/src/+/3192675

**Comment by mxschmitt** [comment]
Relates https://github.com/microsoft/playwright/issues/4077

**Comment by pavelfeldman** [comment]
#### Why was this issue closed?

We are prioritizing the bugs based on the upvotes, recency and our ability to act. It looks like this issue only has a handful of upvotes, has not been touched recently and/or we lack sufficient feedback to act on it. We are closing issues like this one to keep our bug database maintainable. Please feel free to open a new issue and link this one to it if you think this is a mistake.

**Comment by mxschmitt** [comment]
This is fixed in v1.43.
